modules/date.js

```diff
@@ -0,0 +1,6 @@
+import { DateTime } from 'luxon';
+
+const time = document.getElementsByClassName('time');
+const date = DateTime.local();
+const formatDate = date.toFormat("MMMM dd'th 'yyyy', 'hh:mm:ss a");
+time.innerHTML = ` ${formatDate}`;
```

Comment:
- [ ] As shown above the Luxon time isn't responding also.

![mobile](https://github.com/Simpleshaikh1/Awesome-Books/assets/41163571/10c2a402-5954-43cc-8952-a3cee2ab781e)